builds/e2e/templates/longhaul-deploy.yaml

```diff
@@ -35,25 +33,9 @@ parameters:
   test.useTRC: ''
   test.deploymentFileName: ''
   testResultCoordinator.storageAccountConnectionString: ''
+  clientModuleTransportType: ''
 
 steps:
-  - task: CopyFiles@2
-    displayName: 'Copy Edgelet Artifact'
-    inputs:
-      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['edgelet.artifact.name'] }}"
-      TargetFolder: "$(Agent.HomeDirectory)/../artifacts/"
-      CleanTargetFolder: true
-  - task: CopyFiles@2
-    displayName: 'Copy Images Artifact'
-    inputs:
-      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['images.artifact.name'] }}"
-      TargetFolder: "$(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}"
-  - task: CopyFiles@2
-    displayName: 'Copy aziot-identity-service'
-    inputs:
-      SourceFolder: "$(Build.StagingDirectory)"
-      Contents: "$(aziotis.package.filter)"
-      TargetFolder: "$(Agent.HomeDirectory)/../artifacts/"
```

Comment:
I moved this into the `longhaul-setup` template I created as the referenced parameters were already visible there.